tools/js-sdk-release-tools/src/common/codeownersAndignorelink/generateCodeOwnersAndIgnoreLink.ts

```diff
@@ -0,0 +1,98 @@
+import fs from "fs";
+import path from "path";
+import shell from "shelljs";
+import { logger } from "../../utils/logger.js";
+import { SDKType } from "../types.js";
+import { getNpmPackageName } from "../utils.js";
+import { tryGetNpmView } from "../npmUtils.js";
+import { getGeneratedPackageDirectory } from "../../common/utils.js";
+import { posix } from "node:path";
+export async function generateCodeOwnersAndIgnoreLink(
+    sdkType: SDKType,
+    options: {
+        changedPackagePaths: string[];
+        typespecProject?: string;
+        typeSpecDirectory: string;
+        sdkRepo: string;
+    },
+) {
+    if(SDKType.ModularClient === sdkType) {
+        logger.info(`Start with ModularClient.`);
+        const typeSpecDirectory = path.posix.join(
+            options.typeSpecDirectory,
+            options.typespecProject!,
+        );
+        const packageDirectory = await getGeneratedPackageDirectory(
+            typeSpecDirectory,
+            options.sdkRepo.replaceAll("\\", "/"),
+        );
+        const relativePackageDirToSdkRoot = posix.relative(
+            posix.normalize(options.sdkRepo.replaceAll("\\", "/")),
+            posix.normalize(packageDirectory),
+        );
+        await generateCodeOwnersAndIgnoreLinkForPackage(
+            relativePackageDirToSdkRoot,
```

Comment:
this is what you need, could you simplify the generateCodeOwnersAndIgnoreLink's parameters